playbooks/01-bootstrap.yml

```diff
@@ -40,6 +40,7 @@
     - name: Run ci_setup role
       tags:
         - bootstrap
+        - packages
```

Comment:
we're using `import_role` here - the tags associated to this task are applied to all of the imported tasks under `ci_setup` - which isn't what we want to do, since there's not only "package related" tasks in there.

Not sure about the intended effect?

Reply:
I'm trying to resolve some issues in CI jobs using the ci framework. I am trying two paths at the moment.

First solution:
The issue is described here: https://github.com/openstack-k8s-operators/ci-framework/pull/1403 (first attempt to resolve the issue). The issue is occurring in all STF-related jobs, e.g. https://github.com/infrawatch/service-telemetry-operator/pull/589.

Second solution (this PR + [this ](https://github.com/openstack-k8s-operators/ci-framework/pull/1453)+[this](https://github.com/openstack-k8s-operators/ci-framework/pull/1461))
I came to this by trying to skip the install_yamls portion of the setup in bootstrap and infra playbooks:
https://github.com/infrawatch/service-telemetry-operator/pull/590 
This resulted in not having the oc binaries available, which is why I was trying to add the packages tag in this PR.

Reply:
hmm after a quick look, the `oc` comes here:
https://github.com/openstack-k8s-operators/ci-framework/blob/main/roles/ci_setup/tasks/packages.yml#L18-L83

It seems to be missing the `packages` tag (as well as `bootstrap btw`) - I'd start with adding it here first.

Reply:
I think that it is covered with [the always tag in main](https://github.com/openstack-k8s-operators/ci-framework/blob/17b29222d62e8fe786d5bc9156feede80914e21e/roles/ci_setup/tasks/main.yml#L22-L25).

I think all the tasks _should_ be run each time,  unless a tag is specified.  
However, we want this to be run always, so the tasks should be tagged as `always`, right?